public/js/render.js

```diff
@@ -0,0 +1,46 @@
+var renderModule = function () {
```

Comment:
change it to either:

```js
var renderModule = {
  renderEvents: function(...
}
```

or 
```js
var renderEvents = function(...
```

Also the same on the getEvents functions
please please please just do this. @Karyum @MohamedOmarii